src/main/resources/db/migration/V1_0__create_shortened_urls_schema.sql

```diff
@@ -0,0 +1,7 @@
+create table shortened_urls
+(
+    id           text      not null primary key,
+    original_url text      not null unique,
+    title        text      not null,
```

Comment:
I would **make a title optional**, since most people don't care about it. They just need a shortened URL.